client/remote.go

```diff
@@ -86,12 +87,16 @@ func (remote *RemoteVPS) Bootstrap(runner SSHSession, name string, config *Confi
 
 	println("=============================\n")
 
+	repo, err := common.GetLocalRepo()
+	origin, err := repo.Remote("origin")
```

Comment:
Assigning `err` but never using them...

I'm going to tie up this PR - it's been over a month without any activity, @arjansandhu.